packages/actions/src/Call/callProcedure.js

```diff
@@ -65,11 +66,18 @@ export const callProcedure = (client) => async (request) => {
 	})
 	if (errors.length > 0) {
 		const error = /** @type {import('./TevmCallError.js').TevmCallError}*/ (errors[0])
+
+		// Decode the reason if it's a revert
+		let message = error.message
+		if (error.code === 3 || error.message.includes('revert')) {
```

Comment:
```suggestion
		if (error.code === RevertError.code) {
```

Lookat RevertError in the errors pacakge you will notice that it has a static property on it. 

We shouldnt need the message.include I don't think but correct me if I'm wrong

Reply:
does @claude work yet? Hey @claude can you do this. Replace hardcoded reverences to error.code 3 with RevertErrror.code?

Reply:
@claude are you there?